SalesforceSDK/Salesforce.SDK.Core/Source/Settings/SalesforceConfig.cs

```diff
@@ -109,7 +109,7 @@ public virtual string ServerFilePath
 
         public abstract string ApplicationTitle { get; }
 
-        public virtual bool IsApplicationTitleVisible { get { return false; } }
+        public abstract bool IsApplicationTitleVisible { get; }
```

Comment:
Make sure that the sample applications are also updated, they won't build since they're not implementing this property.